azure/app/parameters/development.template.json

```diff
@@ -8,6 +8,9 @@
     "appServiceHostName": {
       "value": "development.additional-teaching-payment.education.gov.uk"
     },
+    "vspAppServiceDockerImage": {
+      "value": "dfedigital/teacher-payments-service-verify:${vspDockerImageTag}"
+    },
```

Comment:
This needs to be specified for `production` and `test` as well, or they will be undeployable.